hphp/runtime/vm/jit/vasm-lower.cpp

```diff
@@ -33,6 +33,52 @@ namespace HPHP { namespace jit {
 
 ///////////////////////////////////////////////////////////////////////////////
 
+namespace detail {
+
+jit::vector<int> computeDominatingConstraints(Vunit& unit) {
+  jit::vector<int> vregConstraints(unit.blocks.size(), INT_MAX);
```

Comment:
`s/INT_MAX/std::numeric_limits<int>::max()/` everywhere.

Or better yet, something like

```
constexpr auto kInvalidConstraint = std::numeric_limits<int>::max();
```

and then use `kInvalidConstraint` everywhere.